### ISSUE TITLE
Running git-seekret config on existing configured repository with enabled rules deletes enabled rules

### ISSUE THREAD

**Issue by alain-hoang** [apuigsech/git-seekret#20]
If `git-seekret` has rules enabled for a repository, running `git-seekret config` will delete the pre-existing rulesets enabled.   This appears to be a bug in the functionality as `git-seekret config` without the `--init` option should not delete configuration information.

Here is an example run to illustrate

```
$ ./git-seekret config
Config:
	version = 1
	rulespath = /path/to/seekret/rules
	rulesenabled = aws.secret_key,aws.access_key,certs.pgp,certs.rsa,certs.generic,password.pwd,password.pass,password.cred,password.password,unix.passwd
	exceptionsfile = 

$ ./git-seekret config
Config:
	version = 1
	rulespath = /path/to/seekret/rules
	rulesenabled = 
	exceptionsfile = 
```

Note that the rulespath is preserved but the rulesenabled does not.  This appears to be related to the querying of enabled rules in SaveConfig()  in `config.go`.   It appears there is a Seekret struct being initialized but its values are empty.

**Comment by alain-hoang** [comment]
I made a [pull request](https://github.com/apuigsech/git-seekret/pull/21) that addresses this behavior.

**Comment by apuigsech** [comment]
Fixed at 744c6089a1571912a9fe77e4df448861622914b9
